client/homebrew/pages/editPage/editPage.jsx

```diff
@@ -3,28 +3,8 @@ require('./editPage.less');
 const React = require('react');
 const createClass = require('create-react-class');
 const _ = require('lodash');
-const request = require('superagent');
-const { Meta } = require('vitreum/headtags');
 
-const Nav = require('naturalcrit/nav/nav.jsx');
-const Navbar = require('../../navbar/navbar.jsx');
-
-const NewBrew = require('../../navbar/newbrew.navitem.jsx');
-const ReportIssue = require('../../navbar/issue.navitem.jsx');
-const PrintLink = require('../../navbar/print.navitem.jsx');
-const Account = require('../../navbar/account.navitem.jsx');
-const RecentNavItem = require('../../navbar/recent.navitem.jsx').both;
-
-const SplitPane = require('naturalcrit/splitPane/splitPane.jsx');
-const Editor = require('../../editor/editor.jsx');
-const BrewRenderer = require('../../brewRenderer/brewRenderer.jsx');
-
-const Markdown = require('naturalcrit/markdown.js');
-
-const googleDriveActive = require('../../googleDrive.png');
-const googleDriveInactive = require('../../googleDriveMono.png');
-
-const SAVE_TIMEOUT = 3000;
+const EditorPage = require('../basePages/editorPage/editorPage.jsx');
 
 const EditPage = createClass({
 	getDefaultProps : function() {
```

Comment:
...(continued) And one thing we can do to streamline, is to make those checks for missing values just *once* somewhere. Even without this PR, we have *multiple* separate components that all have this `getDefaultProps` with a brew, all just in case no brew got passed down. None of that needs to be there, if the highest component in the chain that ever sees the brew does the check, then we *know* all the child components will have a good brew.

Reply:
I was thinking on a similar line - there's not much point to process the brew on the `NewPage` or `EditPage` if we're just going to process it all again in the base `EditorPage`. But I think the other PR we discussed, to implement a base default Brew at a higher level - either in `server.js` or `homebrew.jsx` - that is passed to all Routes will prevent the need to process the brew properties on each Page.